src/tests/setupTests.ts

```diff
@@ -1,4 +1,3 @@
-import 'isomorphic-fetch' // polyfill for fetch
```

Comment:
can we just `import 'whatwg-fetch'` instead? It's currently a dependency, and I'm actually going to use that in SRC in favor of the react-app-polyfill packages since they trigger Jest's memory leak detection.

Reply:
I have no preference.  Updated